README.md

```diff
@@ -8,7 +8,6 @@
   - [🛠 Built With](#built-with)
     - [Tech Stack](#tech-stack)
```

Comment:
- [OPTIONAL] Great work removing the **_Live Demo Section_**. The technologies listed under the Tech Stac section were not used to build this project, especially React.js and Epress.js. Kindly remove them from the README. 👍

Reply:
Removed as per recommendations.
Regards